src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +219,19 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+
+.visEditorSidebar__aggGroup .euiAccordion {
+  // to apply truncated string with ellipsis for agg description
+  & > .euiFlexGroup > .euiFlexItem:first-child,
+  .euiAccordion__buttonContent,
+  // prevent appearing scroll bar in .euiAccordion__childWrapper due to negative margin on some children's EuiFlexGroup
+  .euiForm > .euiFormRow:first-child,
+  [aria-controls="advancedAccordion"] {
```

Comment:
and to prevent scroll bar in Accordion content
<img width="321" alt="bar" src="https://user-images.githubusercontent.com/11524637/61465777-644aa680-a981-11e9-8d62-86bb04dbf254.png">

due to negative margin on EuiFlexGroup (Agg selector label row and Advanced accordion title)

<img width="600" alt="margin" src="https://user-images.githubusercontent.com/11524637/61465786-67459700-a981-11e9-8696-79a1b4188bf5.png">

Reply:
It was fixed in https://github.com/elastic/eui/pull/2143.